x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/cumulative_sum.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FormattedIndexPatternColumn, ReferenceBasedIndexPatternColumn } from '../column_types';
+import { IndexPatternLayer } from '../../../types';
+import { checkForDateHistogram, dateBasedOperationToExpression } from './utils';
+import { OperationDefinition } from '..';
+
+const ofName = (name?: string) => {
+  return i18n.translate('xpack.lens.indexPattern.cumulativeSumOf', {
+    defaultMessage: 'Cumulative sum rate of {name}',
```

Comment:
Are we expecting this to be `Cumulative sum of <field>`? It's currently `Cumulative sum of <reference label>`?

This is part of my reference UI work, so you might not need to make a change. I'm regenerating the default labels whenever you modify an operation.

Reply:
hm, good point, I'm not sure actually. Let's keep that for now and see whether we want to keep it once transition/UI is in place